plugin/scripts/highlight.js

```diff
@@ -205,22 +197,19 @@ const highlight = (function() {
   // sendScreenshot makes the API request to send the screenshot
   // data URI to a tracer with a specific ID.
   async function sendScreenshot(screenshot, id, conf) {
```

Comment:
Now that all configuration is stored in the background page, we don't need `conf`. Can you remove the conf argument here?